packages/build-info/src/frameworks/framework.ts

```diff
@@ -12,6 +12,7 @@ export enum Category {
 }
 
 export enum Accuracy {
+  Forced = 5, // forced framework
```

Comment:
Maybe it would be nice to elaborate a bit on what a forced framework means?

Reply:
This term is used across the codebase to specify if we don't detect the framework instead, it get's forced by the user, for example through specifying the `framework = "astro"` property inside the `netlify.toml`

https://docs.netlify.com/configure-builds/file-based-configuration/#netlify-dev

Reply:
That's great, just saying that having this context in a code comment would probably help our future selves understand what it refers to.

Reply:
Updated the PR with a better comment